src/shared/UI/ProductSection/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { useGetNewProductsQuery, useGetPopularProductsQuery } from "@src/app/api/products";
+import { ProductCard } from "@components";
+import { SeeMoreButton, useTypedTranslation, type ProductSectionUnion, type ProductCardType } from "@shared";
+
+export default function ProductSection ({ section }: { section: ProductSectionUnion; }): JSX.Element {
+  const t = useTypedTranslation();
+  const { data, isLoading, error } = section === "new" ? useGetNewProductsQuery("") : useGetPopularProductsQuery("");
+  if (isLoading || !data || error) return <p>Loading...</p>;
+
+  return (
+    <section className="my-20 w-full">
+      <h5 className="h5 text-center">{section === "new" ? t("newProductSection") : t("popularProductSection")}</h5>
+      <div className="my-12 flex overflow-x-auto overflow-y-hidden px-10 before:m-auto after:m-auto">
```

Comment:
Let's use horisontal scroll and hide scrollbar. It will look good.

Reply:
Here is example i used when i work with pure js. U can use it with TS.

Reply:
[Screencast from 04.09.23 23:29:04.zip](https://github.com/Vladyslav-Samokryk/gift-hub-frontend/files/12516571/Screencast.from.04.09.23.23.29.04.zip)

Reply:
I do the same but in CSS. I see that the scrollbar did not hide in firefox and I fixed it.

Reply:
[Screencast from 05.09.23 22:24:20.webm](https://github.com/Vladyslav-Samokryk/gift-hub-frontend/assets/62620618/113bd491-8e1c-44f9-98d7-942b83aed9bb)
Okey, yes u hidded scrollbar, but now user can not scroll list(only if press 'shift' and then scroll). We can use hook or return back to old version? but add styles for chrome version of scroll.